src/app/api/boosts/claim/route.ts

```diff
@@ -72,8 +72,8 @@ const settings = {
 
 const alchemy = new Alchemy(settings);
 
-async function ownsNFT(userAddress: string, contract: string) {
-    const response = await alchemy.nft.verifyNftOwnership(userAddress, contract);
+async function verifyNftOwnership(userAddress: string, contracts: string[]) {
+    const response = await alchemy.nft.verifyNftOwnership(userAddress, contracts);
```

Comment:
this function can also take an array of contract addresses?

Reply:
yes, it has an overloaded version